apps/backend/src/users/users.service.ts

```diff
@@ -8,16 +8,30 @@ import { UserWithFavoriteDrinks } from './entities/UserWithFavoriteDrinks';
 export class UsersService {
   constructor(readonly prisma: PrismaService) {}
 
+  async update(authSchId: string, gender?: Gender, weight?: number) {
+    return await this.prisma.user.updateMany({
```

Comment:
doesn't really matter, but you can use the update method (instead of updateMany), since it will only ever update one record at a time